Makefile

```diff
@@ -82,6 +82,14 @@ install:
 	install -D -m0755 buildah $(DESTDIR)/$(BINDIR)/buildah
 	$(MAKE) -C docs install
 
+.PHONY: uninstall
+uninstall:
+	rm -f $(DESTDIR)/$(BINDIR)/buildah
+	rm -f $(DESTDIR)/$(BINDIR)/runc
+	rm -f $(PREFIX)/share/man/man1/buildah*.1
+	rm -f $(DESTDIR)/${BASHINSTALLDIR}/buildah
+	rm -r /opt/cni/bin
```

Comment:
I would also leave this alone, runc and /opt/cni/bin can be used by other packages.